code/modules/projectiles/pins.dm

```diff
@@ -39,7 +43,10 @@
 				if(!user.temporarilyRemoveItemFromInventory(src))
 					return .
 				if(gun_insert(user, targetted_gun))
-					balloon_alert(user, "firing pin inserted.")
+					if(old_pin)
+						balloon_alert(user, "swapped firing pin")
+					else
+						balloon_alert(user, "inserted firing pin")
```

Comment:
```suggestion
				balloon_alert(user, "[old_pin ? swapped : inserted] firing pin")
```

probably could do it like this

Reply:
your code doesn't work because it isn't in quotes, but also i think the code before this is more readable imo